tests/phpunit/CRM/Case/XMLProcessor/ProcessTest.php

```diff
@@ -77,17 +81,28 @@ public function testCreateActivityWithDefaultContactByRelationship() {
     $this->activityTypeXml->default_assignee_relationship = $this->relationshipTypeId;
 
     $this->process->createActivity($this->activityTypeXml, $this->params);
-    $this->assertActivityAssignedToContactExists();
+    $this->assertActivityAssignedToContactExists($this->assigneeContactId);
+  }
+
+  public function testCreateActivityWithDefaultContactByRelationButTheresNoRelationship() {
```

Comment:
Can we rename this to `testCreateActivityWithDefaultContactByRelationButThereIsNoRelationship` ?